vassal-app/src/main/java/VASSAL/build/module/PlayerRoster.java

```diff
@@ -719,4 +719,34 @@ protected String translateSide(String side) {
       Resources.getString("Editor.hotkey_label")
     };
   }
+
+
+  /**
+   * {@link VASSAL.search.SearchTarget}
+   * @return a list of any Property Names referenced in the Configurable, if any (for search)
+   */
+  @Override
+  public List<String> getPropertyList() {
+    List<String> l = new ArrayList<>();
+    l.addAll(sides);
+    return l;
```

Comment:
Doesnt compile since sides is already a list. I guess I can just `return sides`, since "supposedly" the things that get these lists don't modify them.